library/core/resource-loader/src/main/java/org/quiltmc/qsl/resource/loader/api/reloader/ResourceReloaderKeys.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.resource.loader.api.reloader;
+
+import net.minecraft.util.Identifier;
+
+/**
+ * This class contains default keys for various Minecraft resource reloaders.
+ *
+ * @see IdentifiableResourceReloader
+ */
+public final class ResourceReloaderKeys {
```

Comment:
Do we want to keep these seperated between client and server? Could be done via two seperate classes or add a Client and server static inner class for this

Reply:
Separated everything into static inner classes.